languages/java/example/Example.java

```diff
@@ -13,40 +14,60 @@ public static void main(String[] args) {
 
         String accessToken = System.getenv("ACCESS_TOKEN");
         UUID organizationId = UUID.fromString(System.getenv("ORGANIZATION_ID"));
+
         String apiUrl = System.getenv("API_URL");
         String identityUrl = System.getenv("IDENTITY_URL");
+        String stateFile = System.getenv("STATE_FILE");
 
         // Configuring the URLS is optional, remove them to use the default values
         BitwardenSettings bitwardenSettings = new BitwardenSettings();
         bitwardenSettings.setApiUrl(apiUrl);
         bitwardenSettings.setIdentityUrl(identityUrl);
 
         try (BitwardenClient client = new BitwardenClient(bitwardenSettings)) {
-            client.accessTokenLogin(accessToken);
+            client.auth().loginAccessToken(accessToken, stateFile);
 
-            ProjectResponse project = client.projects().create(organizationId, "Test Project");
-            System.out.println("Project id: " + project.getID());
+            ProjectResponse project = client.projects().create(organizationId, "Test Project from Java SDK");
+            System.out.println("Project CREATE, id: " + project.getID());
 
             project = client.projects().get(project.getID());
+            System.out.println("Project GET, id: " + project.getID());
 
             ProjectsResponse projects = client.projects().list(organizationId);
-            System.out.println("Projects count: " + projects.getData().length);
+            System.out.println("Projects LIST, count: " + projects.getData().length);
 
-            client.projects().update(project.getID(), organizationId, "Updated Test Project");
+            client.projects().update(organizationId, project.getID(), "Updated Test Project");
+            project = client.projects().get(project.getID());
+            System.out.println("Project UPDATE, new name: " + project.getName());
 
-            SecretResponse secret = client.secrets().create("Secret Key", "Secret Value", "Secret Note",
-                organizationId, new UUID[]{project.getID()});
-            System.out.println("Secret id: " + secret.getID());
+            SecretResponse secret = client.secrets().create(organizationId, "Secret Key", "Secret Value", "Secret Note", new UUID[]{project.getID()});
+            System.out.println("Secret CREATE, id: " + secret.getID());
 
             secret = client.secrets().get(secret.getID());
+            System.out.println("Secret GET, id: " + secret.getID());
 
             SecretIdentifiersResponse secrets = client.secrets().list(organizationId);
-            System.out.println("Secrets count: " + secrets.getData().length);
+            System.out.println("Secrets LIST, count: " + secrets.getData().length);
+
+            client.secrets().update(organizationId, secret.getID(), "Updated Key", "Updated Value", "Updated Noye", new UUID[]{project.getID()});
+            secret = client.secrets().get(secret.getID());
+            System.out.println("Secret UPDATE, new name: " + secret.getKey());
```

Comment:
⛏️ 
```suggestion
            System.out.println("Secret UPDATE, new key: " + secret.getKey());
```

Reply:
For these I picked `name` for readability, but `key` is fine too, especially because that matches the function names.

Updated in https://github.com/bitwarden/sdk/pull/1005/commits/16484df6301a14ca79dc7375907576051d3d720a